Felienne/hedy
# Fix & update "Unquoted Text" errormessage for German

**Description**

- Fixes a typo & the grammar and in the German translation of the "Unquoted Text" error message.
- Updates the error message to contain the same information as the current corresponding English one.

**Fix for**

(No issue filed.)

**How to test**

1. Select "Deutsch" in the language switcher (when logged out) or choose it as your language in your profile (when logged in)
2. Go to ([Level 5](https://www.hedycode.com/hedy/5)
3. Run the following program:
   ```
   name is ask 'Wie heisst du?'
   if name is Hedy print Toll! else print 'Blöd'
   ```
4. Check the error message that appears

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
